dbms/tests/queries/0_stateless/00837_minmax_index.reference

```diff
@@ -0,0 +1,6 @@
+0	5	4.7	6.50	cba	b	2014-01-04
+1	5	4.7	6.50	cba	b	2014-03-11
+11	5	4.7	6.50	cba	b	2014-06-11
+12	5	4.7	6.50	cba	b	2015-01-01
+		"rows_read": 4,
+		"rows_read": 2,
```

Comment:
There is 2 because KeyCondition does not drop out one granule.
https://gist.github.com/nikvas0/528eea3bf556317fb699bd5994e680b4